map/search_mark.cpp

```diff
@@ -167,7 +167,15 @@ class SearchMarkTypeChecker
       {{"shop", "toys"},             SearchMarkType::ShopToys},
       {{"tourism", "theme_park"},    SearchMarkType::ThemePark},
       {{"leisure", "water_park"},    SearchMarkType::WaterPark},
-      {{"tourism", "zoo"},           SearchMarkType::Zoo}
+      {{"tourism", "zoo"},           SearchMarkType::Zoo},
+      {{"tourism", "hotel"},         SearchMarkType::Hotel},
```

Comment:
@vng why it wasn't done before in MM? What was the value to process hotels differently?

Reply:
Was a separate SetHotelType function, but the new approach is better.